package.json

```diff
@@ -24,7 +24,7 @@
     "@rjsf/validator-ajv8": "5.5.0",
     "autosuggest-highlight": "3.3.4",
     "buffer": "6.0.3",
-    "check-dependencies": "1.1.0",
+    "check-dependencies": "^1.1.0",
```

Comment:
You can remove this. node.js has a tendency to add these `^` to resolve conflicts but they're not needed here.